src/cartservice/src/cartstore/RedisCartStore.cs

```diff
@@ -16,95 +16,18 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Grpc.Core;
-using StackExchange.Redis;
+using Microsoft.Extensions.Caching.Distributed;
 using Google.Protobuf;
 
 namespace cartservice.cartstore
 {
     public class RedisCartStore : ICartStore
     {
-        private const string CART_FIELD_NAME = "cart";
-        private const int REDIS_RETRY_NUM = 30;
+        private readonly IDistributedCache _cache;
```

Comment:
The `ConnectionMultiplexer` implementation was the original way to interact with redis database, now the best practice and the guidance is to use the `IDistributedCache` interface and associated objects.